units/XNB2305/XNB2305_script.lua

```diff
@@ -1,18 +1,19 @@
--- T3 nuke launcher
 local NStructureUnit = import('/lua/nomadsunits.lua').NStructureUnit
 local StrategicMissileWeapon = import('/lua/nomadsweapons.lua').StrategicMissileWeapon
 local TacticalMissileWeapon1 = import('/lua/nomadsweapons.lua').TacticalMissileWeapon1
 
+--- Tech 3 Strategic Launcher (Nuke Launcher)
```

Comment:
The parentheses can be confused as the unit name. FAF repo unit script file descriptions use "Strategic Missile Launcher".
```suggestion
--- Tech 3 Strategic Missile Launcher
```